src/cms/urls.py

```diff
@@ -938,6 +938,45 @@
                         ]
                     ),
                 ),
+                url(
+                    r"^user_settings/",
+                    include(
+                        [
+                            url(
+                                r"^$",
+                                settings.UserSettingsView.as_view(),
+                                name="user_settings",
+                            ),
+                            url(
+                                r"^mfa/",
+                                include(
+                                    [
+                                        url(
+                                            r"^authenticate/$",
+                                            settings.AuthenticateModifyMfaView.as_view(),
+                                            name="authenticate_modify_mfa",
+                                        ),
+                                        url(
+                                            r"^get_challenge/$",
+                                            settings.GetMfaChallengeView.as_view(),
+                                            name="get_mfa_challenge",
+                                        ),
+                                        url(
+                                            r"^register/$",
+                                            settings.RegisterUserMfaKeyView.as_view(),
+                                            name="register_new_mfa_key",
+                                        ),
+                                        url(
+                                            r"^delete/(?P<key_id>\d+)$",
+                                            settings.DeleteUserMfaKeyView.as_view(),
+                                            name="delete_mfa_key",
+                                        ),
+                                    ]
+                                ),
+                            ),
```

Comment:
I think in the moment we do not need this block, because the 2FA button of the user settings for now points to `/user_settings/mfa/register` and not to `{region_slug}/user_settings/mfa/register`
You can see this in href of the following link tag, where region slug is not included:
https://github.com/Integreat/integreat-cms/blob/ed22b124c22912e78bc6cc6e4c6eff1e9635cff1/src/cms/templates/settings/user.html#L118
And because 2FA is not part of the issue, I think we can just skip this block